ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py

```diff
@@ -1380,3 +1380,37 @@ def test_paddle_rsqrt(
         frontend=frontend,
         on_device=on_device,
     )
+
+
+# log1p
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="paddle.to_tensor",
+    method_name="log1p",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
Kindly try to use either the "log" safety factor with a number of like `12` or something, a simple safety factor of `2` does not make any difference, the test is still failing for torch for `input=[array(1.e+307)]`